### PR TITLE
Xiaomi MiBeacon/BLE support

### PR DESCRIPTION
**Pull request by Jc2k**
## Proposed change

Document MiBeacon / Xiaomi BLE support.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/75618
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3547
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by Ernst79** [comment]
We might need to add a comment that encrypted sensors aren’t supported yet.

File: source/_integrations/xiaomi_ble.markdown [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  - '@Jc2k'
+  - '@Ernst79'
+ha_domain: xiaomi_ble
+ha_config_flow: true
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+Integrates devices that implement the Xiaomi Mijia BLE MiBeacon protocol and other Xiaomi BLE devicres that support passive collection. It listens to Bluetooth broadcasts that the device makes by itself, allowing us to track the latest sensor values without needing to wake it up from deep sleep to poll and conserving its battery power.
```
Comment (MartinHjelmare) [inline source/_integrations/xiaomi_ble.markdown]:
```suggestion
Integrates devices that implement the Xiaomi Mijia BLE MiBeacon protocol and other Xiaomi BLE devices that support passive collection. It listens to Bluetooth broadcasts that the device makes by itself, allowing us to track the latest sensor values without needing to wake it up from deep sleep to poll and conserving its battery power.
```

File: source/_integrations/xiaomi_ble.markdown [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+The integration will automatically discover devices once the [Bluetooth](/integrations/bluetooth) integration is enabled and functional.
+
+## Supported device classes
```
Comment (MartinHjelmare) [inline source/_integrations/xiaomi_ble.markdown:25]: I'd put this section below the config flow template that tells the user how to set up the integration.

Comment (Jc2k) [reply source/_integrations/xiaomi_ble.markdown:25]: I've done that in last commit, cheers!

**Review by MartinHjelmare — DISMISSED** [review]
Thanks!

**Comment by Jc2k** [comment]
> We might need to add a comment that encrypted sensors aren’t supported yet.

I should have time to finish https://github.com/home-assistant/core/pull/75677 before beta, but can do a follow up if i fail.

**Review by MartinHjelmare — DISMISSED** [review]

File: source/_integrations/xiaomi_ble.markdown [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+title: Xiaomi BLE
+description: Instructions on how to integrate Xiaomi BLE devices into Home Assistant.
+ha_category:
+  - Sensor
+ha_zeroconf: true
```
Comment (MartinHjelmare) [inline source/_integrations/xiaomi_ble.markdown]:
```suggestion
ha_bluetooth: true
```

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**